### PR TITLE
Errors are now properly handled in build script

### PR DESCRIPTION
**Pull request by brandonmaul**
Solves #449

Solves issue where building from source would hang indefinitely

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 DEST := /usr/local/bin/vapor
 
 build:
+        swift build -c release
```
Comment (0xTim) [inline Makefile:4]: Is this needed? Seems a bit redundant

Comment (brandonmaul) [reply Makefile:4]: I found that if I just built it without the release flag, it defaulted to debug.

Which ultimately changed the output folder.

Comment (0xTim) [reply Makefile:4]: Right, sorry I mean why are we doing a build here at all? The `build.swift` script is the one doing the build

Comment (brandonmaul) [reply Makefile]: Ah! Duh. Apologies, should be fixed up now.

**Review by 0xTim — APPROVED** [review]
Thanks!

**Pull request merged**